locales/templates/app.pot

```diff
@@ -2384,3 +2423,48 @@ msgctxt "travel_search_engine_fxlite"
 msgid "See results in %1$s Travel Discovery"
 msgstr ""
 
+#. Dialog
+#. dialog title for introducing Firefox Lite Travel Discovery
+msgctxt "travel_dialog_1_title"
+msgid "Discover More!"
+msgstr ""
+
+#. dialog description for introducing the feature. The %1$s here is Firefox
+#. Lite.
+#, c-format
+msgctxt "travel_dialog_1_description"
+msgid "Explore things to do, accomodations, and more in %1$s Travel Discovery."
+msgstr ""
+
+#. Action button for seeing the search result in Firefox Lite Travel Discovery
```

Comment:
The comment doesn't seem to correspond to the string, unless I'm missing something?

Reply:
This is a dialog (prompt) for first time users. I think the comment makes sense to me.

I do see an error here though,
need to change "accomodations" to "accommodations"
@Conjuror

Reply:
DONE